templates/CMakeLists.txt.template

```diff
@@ -196,18 +196,24 @@
         message(WARNING "gRPC_PROTOBUF_PROVIDER is \"module\" but PROTOBUF_ROOT_DIR is wrong")
     endif()
   elseif("<%text>${gRPC_PROTOBUF_PROVIDER}</%text>" STREQUAL "package")
-    find_package(protobuf CONFIG)
-    if(protobuf_FOUND)
+    find_package(Protobuf)
```

Comment:
hm, looks like I got warnings when doing that:
CMake Warning at CMakeLists.txt:187 (find_package):
  By not providing "FindPROTOBUF.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "PROTOBUF",
  but CMake did not find one.

  Could not find a package configuration file provided by "PROTOBUF" with any
  of the following names:

    PROTOBUFConfig.cmake
    protobuf-config.cmake

I think the proper name of the package is Protobuf, this is also what Konstantin mentioned in his comments